ws/DiscordWebsocket.ts

```diff
@@ -211,7 +211,7 @@ export default class DiscordWebsocket {
       case GATEWAY_OPCODES.DISPATCH: this._onEvent(p); break;
       case GATEWAY_OPCODES.HEARTBEAT: this._heartbeat(); break;
       case GATEWAY_OPCODES.RECONNECT: this.restart(); break;
-      case GATEWAY_OPCODES.INVALID_SESSION: this._identify(); break;
+      case GATEWAY_OPCODES.INVALID_SESSION: setTimeout(() => this._identify, IDENTIFY_TIMEOUT); break;
```

Comment:
That's fair to wait the identify timeout here eventually?
```suggestion
      case GATEWAY_OPCODES.INVALID_SESSION: setTimeout(this._identify, IDENTIFY_TIMEOUT); break;
```

Reply:
Can we not immediately identify?

Reply:
I guess if we get invalid session straight after an identify?

Reply:
I did not tried to immediately identify but I followed https://discord.com/developers/docs/topics/gateway#resuming-example-gateway-resume